locale/eo.js

```diff
@@ -36,25 +36,25 @@
         calendar : {
             sameDay : '[Hodiaŭ je] LT',
             nextDay : '[Morgaŭ je] LT',
-            nextWeek : 'dddd [je] LT',
+            nextWeek : 'dddd[n je] LT',
             lastDay : '[Hieraŭ je] LT',
-            lastWeek : '[pasinta] dddd [je] LT',
+            lastWeek : '[pasintan] dddd[n je] LT',
             sameElse : 'L'
         },
         relativeTime : {
             future : 'post %s',
             past : 'antaŭ %s',
-            s : 'sekundoj',
+            s : 'kelkaj sekundoj',
             ss : '%d sekundoj',
-            m : 'minuto',
+            m : 'unu minuto',
             mm : '%d minutoj',
-            h : 'horo',
+            h : 'unu horo',
             hh : '%d horoj',
-            d : 'tago',//ne 'diurno', ĉar estas uzita por proksimumo
+            d : 'unu tago',//ne 'diurno', ĉar estas uzita por proksimumo
             dd : '%d tagoj',
-            M : 'monato',
+            M : 'unu monato',
             MM : '%d monatoj',
-            y : 'jaro',
+            y : 'unu jaro',
```

Comment:
Adding the word "unu" (=one) because it's about a quantity, and not about an indefinite thing introduced by the article a/an.